src/app/queries/fetchHelpers.ts

```diff
@@ -29,34 +32,33 @@ export const authorizedFetch = async <T>(
         Authorization: `Bearer ${fetchContext.currentUser.access_token}`,
         ...extraHeaders,
       },
+      method,
+      ...(data &&
+        method !== 'get' && {
+          body: JSON.stringify(data),
+        }),
     });
     if (response.ok && response.json) {
       return response.json();
     } else {
       throw response;
     }
-  } catch (error) {
+  } catch (error: unknown) {
     checkExpiry(error, history);
     throw error;
   }
 };
 
-export const useAuthorizedFetch = <T>(url: string): QueryFunction<T> => {
+export const useAuthorizedFetch = <T>(
+  url: string
+  // extraHeaders: RequestInit['headers'] = {},
```

Comment:
Do we need to keep those commented lines?

Reply:
Nope! I'll remove them